cluster/images/hyperkube/Makefile

```diff
@@ -19,7 +19,7 @@
 
 REGISTRY?=gcr.io/google_containers
 ARCH?=amd64
-TEMP_DIR:=$(shell mktemp -d)
+TEMP_DIR:=$(shell mktemp -d 2>/dev/null || mktemp -d -t hyperkube)
```

Comment:
Does not work on linux for me:

```
[root@6f4669653875 /]# mktemp -d -t hyperkube
mktemp: too few X's in template 'hyperkube'
```

This one ```mktemp -d -t hyperkubeXXXXXX``` works on both though, though the XXX part is present on OSX in the resulting file, e.g.:

```
mktemp -d -t hyperkubeXXXXXX
/var/folders/c_/jc02k3ls0sggs3hc8h4qltyxj_5h0_/T/hyperkubeXXXXXX.mFkrScke
```

I'll just change to this one then, cause it's probably not a big deal.